### PR TITLE
ENH: Add SolenoidPair and update GasManifold/Selector code

### PR DESCRIPTION
**Pull request by ZryletTC**
- Adds SolenoidPair class for quick sample changes in RIX
- Changes GasManifold code because new valves are normally open
- Make GasManifold valves easier to use
- Add variables/PVs for Selector sample names and GasManifold valve names

### CODE REVIEW

File: sdsPLC/SDSPLC/DUTs/ST_Strings.TcDUT [hunk 1]
```diff
@@ -0,0 +1,71 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<TcPlcObject Version="1.1.0.1" ProductVersion="3.1.4022.18">
+  <DUT Name="ST_Strings" Id="{732f2c27-82d7-468b-9b70-d5e6f1796e6b}">
+    <Declaration><![CDATA[TYPE ST_Strings :
+STRUCT
+	(* Twelve strings used to store sample names *)
+	{attribute 'pytmc' := '
```
Comment (klauer) [inline sdsPLC/SDSPLC/DUTs/ST_Strings.TcDUT:7]: You could have also done an `ST_String` with a single pragma'd `STRING` element.
Then you'd create a `ARRAY OF ST_String` with 12 strings. pytmc and twincat-ads would support this just fine.

File: sdsPLC/SDSPLC/POUs/GasManifold/FB_GasManifold.TcPOU [hunk 2]
```diff
@@ -1,5 +1,5 @@
 ﻿<?xml version="1.0" encoding="utf-8"?>
-<TcPlcObject Version="1.1.0.1" ProductVersion="3.1.4024.2">
+<TcPlcObject Version="1.1.0.1" ProductVersion="3.1.4022.18">
```
Comment (klauer) [inline sdsPLC/SDSPLC/POUs/GasManifold/FB_GasManifold.TcPOU:2]: You jumped back in version from 4024 to 4022, perhaps by mistake?

Comment (klauer) [reply sdsPLC/SDSPLC/POUs/GasManifold/FB_GasManifold.TcPOU:2]: I'd also recommend getting the pre-commit settings installed for the repo to help with style. It'd be a good thing to do in a follow-up PR to not hide the real content changes here, though.

File: sdsPLC/SDSPLC/DUTs/ST_ManifoldValve.TcDUT [hunk 3]
```diff
@@ -30,6 +30,12 @@ STRUCT
      '}
 	xILK :	BOOL; // permissive bit
 
+	{attribute 'pytmc' := '
+        pv: Name
+        io: i
+    '}
+    sName: STRING;
```
[the remark this answers is not in the document]
Comment (ZryletTC) [reply sdsPLC/SDSPLC/DUTs/ST_ManifoldValve.TcDUT:37]: Yeah, this is a weird situation where I have the multiple instances of the same PLC/project. Can ads-deploy handle that? Until now, I've always just build them each separately from psbuild using pytmc.

File: sdsPLC/SDSPLC/DUTs/ST_Strings.TcDUT (hunk 1 above)
Comment (ZryletTC) [reply sdsPLC/SDSPLC/DUTs/ST_Strings.TcDUT:7]: Ahh that makes a bit more sense 🚀 
I'll implement this and and other cleanup stuff in the next PR.

File: sdsPLC/SDSPLC/DUTs/ST_ManifoldValve.TcDUT (hunk 3 above)
[the remark this answers is not in the document]
Comment (klauer) [reply sdsPLC/SDSPLC/DUTs/ST_ManifoldValve.TcDUT:37]: I haven't tested it, but I believe you could still use ads-ioc here - unless SDS has some additional module requirements, of course.

What you'd do is:
1. Use ads-deploy to create a single iocBoot directory
2. Rename the generated iocBoot directory + modify the generated `Makefile` (set the DB parameters as needed)
3. Copy/paste that boot directory and repeat the renaming + PV prefix setting process as needed

[the remark this answers is not in the document]
Comment (ZryletTC) [reply sdsPLC/SDSPLC/DUTs/ST_ManifoldValve.TcDUT:37]: Yeah, I'm essentially doing that on psbuild already, I guess there's no reason to not do it from plcprog and check it in

**Review by slacAWallace — APPROVED** [review]
These changes look fine to me. I also agree with Ken's comments.

**Pull request merged**